### DISCUSSION TITLE
Font_family don't work for me with google fonts

### DISCUSSION THREAD

**Discussion by franvillalba72** [reflex-dev/reflex#2077] [General]
Hi everyone,
the font_family prop doesn't work for me when put a google font.
I don't know how to put a google font in my app.

Thanks.

**Comment by morawer** [comment]
Hi Fran,

I have the same problem.

**Comment by Alek99** [comment]
Hi are you also attaching a style sheet with the google font?

See how we do it in Reflex web:
https://github.com/reflex-dev/reflex-web/blob/2ff5da2b56106b70a8283286d8fab51cf908f1d9/pcweb/styles/styles.py#L68-L70

```python
STYLESHEETS = [
    "https://fonts.googleapis.com/css2?family=Instrument+Sans:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&family=IBM+Plex+Mono:ital,wght@0,500;0,600;1,600&display=swap",
]
```
https://github.com/reflex-dev/reflex-web/blob/2ff5da2b56106b70a8283286d8fab51cf908f1d9/pcweb/pcweb.py#L16

```python
from pcweb import styles
...

# Create the app.
app = rx.App(
    stylesheets=styles.STYLESHEETS,
    ...
)
```

**Reply by morawer** [reply]
Thank you @Alek99. Problem solved.

**Reply by franvillalba72** [reply]
It works. Thank you @Alek99 .

**Comment by DiegoHC06** [comment]
i did was download and install the font it on my pc, now i can visualize
